packages/blade/src/components/Card/types.ts

```diff
@@ -0,0 +1,18 @@
+import type { BaseBoxProps } from '~components/Box/BaseBox';
+
+type CardRootProps = {
+  isSelected?: boolean;
+  isFocused?: boolean;
+  scaleOnHover?: boolean;
+  onClick?: () => void;
+  children?: React.ReactNode;
+  href?: string;
+  as?: 'label';
+  accessibilityLabel?: string;
+} & BaseBoxProps;
+
+type LinkOverlayProps = {
+  href?: string;
```

Comment:
can we have all the link props here?

Reply:
Already added href, target, onClick i think only one is left is rel. 

Will add that.